code-studio/src/js/initApp/initApp.js

```diff
@@ -1,6 +1,8 @@
 // TODO (brent) - way too many globals
 /* global script_path, CDOSounds, dashboard, appOptions, trackEvent, Applab, Blockly, ga*/
 import $ from 'jquery';
+import { PlayZone } from '../components/playzone';
```

Comment:
If you do export default class, then this can just be `import PlayZone from ...`